pyomegle/pyomegle.py

```diff
@@ -87,7 +88,7 @@ def __init__(self, events_handler, rcs=1, firstevents=1, spid='', random_id=None
 
     def _randID(self, length):
         """ Generates a random ID for chat session """
-        return ''.join([random.choice('0123456789ABCDEFGHJKLMNPQRSTUVWXYZ')
+        return ''.join([random.choice('23456789ABCDEFGHJKLMNPQRSTUVWXYZ')
```

Comment:
Why you removed `01`?

Reply:
After doing some delving into the Omegle.js source on their website I found that their random ID generation has changed, they no longer use 0 or 1 in their random ID's.

Here's the sectionfrom omegle.js...
https://www.omegle.com/static/omegle.js?649
```javascript
var randID=Cookie.read("randid");
randID&&8===randID.length||(randID=function(){for(var a="23456789ABCDEFGHJKLMNPQRSTUVWXYZ",b="",c=0;8>c;c++){var d=Math.floor(Math.random()*a.length);
```


I thought it best to match their style of doing so. There's a chance that RandomID's including certain sequences of 1's or 0's might be rejected by their systems.

Reply:
Apologies for not explaining this in the initial commit, it must have escaped me. But I hope this helps. :)